packages/forge/blocks/zendeskChat/actions/authenticate.ts

```diff
@@ -0,0 +1,50 @@
+import { createAction, option } from '@typebot.io/forge'
+import jwt from 'jsonwebtoken';
+import { auth } from '../auth'
+
+export const authenticate = createAction({
+  name: 'Authenticate User',
+  auth,
+  options: option.object({
+    userId: option.string.layout({
+      label: 'User ID',
+      isRequired: true,
+    }),
+    name: option.string.layout({
+      label: 'Name',
+      isRequired: true,
+    }),
+    email: option.string.layout({
+      label: 'Email',
+      isRequired: true,
+    }),
+    emailVerified: option.string.layout({
+      label: 'Email Is Verified',
+      defaultValue: "false"
+    }),
```

Comment:
```suggestion
    wasEmailVerified: option.boolean.layout({
      label: 'Email was verified',
    }),
```

I think it is clearer

Reply:
Email Verification is a status of the email, so it's probably better to describe it in terms of the current state. How about 'isEmailVerified' and keep the label the same?

Reply:
Ok, but still convert it to boolean and non-title case?

Reply:
@baptisteArno a question on the boolean. If I change it to boolean, it doesn't appear that I can set that value from a variable, which I would need to be able to do as we would be tracking as part of a user profile. Am I missing something? Otherwise, I think we need to leave it as a string so we can do that.